clang/lib/CIR/CodeGen/CIRGenBuilder.h

```diff
@@ -434,6 +439,35 @@ class CIRGenBuilderTy : public mlir::OpBuilder {
     return Address(baseAddr, ptrTy, addr.getAlignment());
   }
 
+  // FIXME(cir): CIRGenBuilder class should have an attribute with a reference
+  // to the module so that we don't have search for it or pass it around.
+  // FIXME(cir): Track a list of globals, or at least the last one inserted, so
+  // that we can insert globals in the same order they are defined by CIRGen.
```

Comment:
Nitpick: you are mixing comments with "///" with "//", just pick one for this, or leave an empty line in between!